packages/dds/tree/src/domains/nodeKey/nodeKeySchema.ts

```diff
@@ -26,17 +20,6 @@ const builder = new SchemaBuilderInternal({ scope: "com.fluidframework.nodeKey"
 export const nodeKeyTreeSchema = builder.leaf("NodeKey", ValueSchema.String);
 assert(nodeKeyTreeSchema.name === nodeKeyTreeIdentifier, 0x7ae /* mismatched identifiers */);
```

Comment:
We can get rid of the rest of the things in this file too, I think. In fact, I don't see why we shouldn't delete the entire `src/domains/nodeKey` folder.